### PR TITLE
Update release checklist

### PR DESCRIPTION
**Pull request by nikola-matic**
This is an ongoing list of things we'll be changing in the release, and will be merged once we're satisfied with it.

### CODE REVIEW

File: ReleaseChecklist.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ At least a day before the release:
        Make sure it contains four binaries: ``solc-windows.exe``, ``solc-macos``, ``solc-static-linux`` and ``soljson.js``.
  - [ ] Take the ``solc-bin-binaries.tar`` tarball from ``c_release_binaries`` run of the tagged commit in circle-ci and add all binaries from it to solc-bin.
  - [ ] Run ``npm run update -- --reuse-hashes`` in ``solc-bin`` and verify that the script has updated ``list.js``, ``list.txt`` and ``list.json`` files correctly and that symlinks to the new release have been added in ``solc-bin/wasm/`` and ``solc-bin/emscripten-wasm32/``.
+       **Note: you should run `npm install` prior to the above step, and must run it if you've got a clean checkout of the solc-bin repo.**
```
Comment (cameel) [inline ReleaseChecklist.md]: Well, if you really want it here, then it should be a separate step. And *before* `npm run update` :)

**Comment by cameel** [comment]
Pinging @nikola-matic. I guess would be best to merge this *before* we start the release :) And it needs only minor adjustments.

**Comment by nikola-matic** [comment]
Pinging @cameel to review and approve.

File: ReleaseChecklist.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ At least a day before the release:
  - [ ] Take the ``github-binaries.tar`` tarball from ``c_release_binaries`` run of the tagged commit in circle-ci and add all binaries from it to the release page.
        Make sure it contains four binaries: ``solc-windows.exe``, ``solc-macos``, ``solc-static-linux`` and ``soljson.js``.
  - [ ] Take the ``solc-bin-binaries.tar`` tarball from ``c_release_binaries`` run of the tagged commit in circle-ci and add all binaries from it to solc-bin.
+ - [ ] Run `npm install` if you've got a clean checkout of the solc-bin repo.
```
Comment (cameel) [inline ReleaseChecklist.md]:
```suggestion
 - [ ] Run ``npm install`` if you've got a clean checkout of the solc-bin repo.
```

**Review by cameel — DISMISSED** [review]
Needs one more tweak for consistency with the weird double backticks used in the document :)

**Review by cameel — APPROVED** [review]

**Pull request merged**